Cargo.toml

```diff
@@ -24,7 +24,11 @@ exclude       = [
 publish = false
 
 [features]
-default = []
+os-poll = []
```

Comment:
I think it would be nice to have the purpose/contents of each feature documented. Newcomers, as myself, will appreciate it. Not a strong requirement, just a nice to have, if you have time.

Reply:
Yep that makes sense. I'll add some explanation!

Reply:
Thanks!